initializr/src/main/resources/templates/home.html

```diff
@@ -11,174 +11,167 @@
-                    <% it.content.each { %>
-                    <div class="checkbox" data-range="${it.versionRange}">
-                        <label>
-                            <input type="checkbox" name="style" value="${it.id}">
-                            ${it.name}
-                        </label>
-                    </div><% } %>
+                <div class="form-group col-md-6 col-sm-12">
+                    <div id="starters">
+                    </div>
                 </div>
-                <% } %>
             </div>
-        </div>
-        <div class="row">
-            <p class="text-center">
-                <button name="generate-project" type="submit" class="btn btn-lg btn-primary" role="button">
-                    <span class="glyphicon glyphicon-download"></span> Generate Project
-                </button>
-            </p>
-        </div>
-        <div class="page-header top10">
-            <h1>Spring CLI <small>Quickly prototype with Spring</small></h1>
-        </div>
-        <div class="row">
-            <div class="col-sm-offset-1 col-sm-5">
-                <h3>Spring CLI Zip</h3>
-                <p>
-                    <a href="/spring.zip" class="btn btn-lg btn-primary" role="button">
-                        <span class="glyphicon glyphicon-download-alt"></span> Download Spring CLI Zip
-                    </a>
-                </p>
-                <p>
-                    Download the spring CLI as a zip distribution (unpack and run bin/spring on a command line).
+            <div class="row">
+                <p class="text-center">
+                    <button name="generate-project" type="submit" class="btn btn-lg btn-primary generate" role="button">
+                        <span class="glyphicon glyphicon-download"></span> Generate Project
+                    </button>
                 </p>
             </div>
-            <div class="col-sm-5">
-                <h3>Spring CLI Installer</h3>
-                <p>
-                    <a href="/install.sh" class="btn btn-lg btn-primary" role="button">
-                        <span class="glyphicon glyphicon-download-alt"></span> Download Spring Installer
-                    </a>
+            <div class="row">
+
+                <div id="starters-list" class="page-header">
+                    <h2>${dependencies.title}</h2>
+                </div>
+
+                <div id="dependencies">
+
+                    <% dependencies.content.each { %>
+                    <div class="form-group col-sm-6">
+                        <h3>${it.name}</h3>
+                        <% it.content.each { %>
+                        <div class="checkbox" data-range="${it.versionRange}">
+                            <label>
+                                <input type="checkbox" name="style" value="${it.id}">
+                                ${it.name}
+                                <p class="help-block">${it.description}</p>
```

Comment:
I think that many dependencies' descriptions aren't really helpful, we should improve that.
Also, I'd like to remove the `Support for` prefix for all of them.